src/gui/rss/rsswidget.cpp

```diff
@@ -79,23 +79,19 @@ RSSWidget::RSSWidget(IGUIApplication *app, QWidget *parent)
     m_ui->rssDownloaderBtn->setIcon(UIThemeManager::instance()->getIcon(u"downloading"_s, u"download"_s));
 #endif
 
-    m_articleListWidget = new ArticleListWidget(m_ui->splitterMain);
```

Comment:
Just for fun. It would be enough to just change this line to the next one (and the same for `m_feedListWidget`):
```c++
m_articleListWidget = m_ui->articleListWidget;
```